gating/pre_merge_test/run_deploy_mnaio.sh

```diff
@@ -38,6 +38,9 @@ export RE_JOB_TRIGGER="${RE_JOB_TRIGGER:-PR}"
 export RE_HOOK_ARTIFACT_DIR="${RE_HOOK_ARTIFACT_DIR:-/tmp/artifacts}"
 export RE_HOOK_RESULT_DIR="${RE_HOOK_RESULT_DIR:-/tmp/results}"
 
+## Functions -----------------------------------------------------------------
+source /opt/rpc-openstack/scripts/functions.sh
```

Comment:
Is there a reason this can't go in front of the RE stuff?

Reply:
Not really, happy to move it if it's a concern.